src/GT.Core/Services/Impl/GTCompanyService.cs

```diff
@@ -64,26 +65,87 @@ public async Task<CompanyDTO> AddAsync(CompanyDTO dto)
 			}
 		}
 
-		public string AddCompanyLogo(byte[] file)
+		public async Task AddCompanyLogo(CompanyLogoDTO companyLogoDTO)
 		{
-			throw new NotImplementedException();
+			//create folder if not exist
```

Comment:
Suggested change: 
`//Create folder if it does not exist`

Reply:
Better, I've changed to the suggested comment